scripts/sct_segment_graymatter.py

```diff
@@ -53,22 +53,18 @@
 import matplotlib
 matplotlib.use('Agg')
 
-
-import sct_maths
-import sct_process_segmentation
-import sct_register_multimodal
 from msct_gmseg_utils import (apply_transfo, average_gm_wm, binarize,
```

Comment:
Why would these msct_ and sct_ imports still be here?

If the file has too many functions, just skip it do it for now.